NEWS.md

```diff
@@ -1,16 +1,78 @@
 next ():
 
-- Add a NEWS file
-- Sort alerts within a category by time, newest first (including node-to-node messages).
-- Optimize CSS to load everything but the first theme in parallel
-- Tighten security (Remove DSA and compat code).
-- Only update essential plugins if we have to.
-- Improve translations.
-- Show peer locations, not distance in peer location histogram.
-- Add much more CSS support to the content filter.
-- Fix problems with blank bookmark names. Thanks to _xiaoyu for the report!
-- Fix missing the software category in bookmarks.
-- Fix caching of static assets
+- switch to gradle with witness as build system
+- run in background mode
+- switch from jni to jna
+- Override list request identifier
+- use fallocate
+- ipv6 fixes
+- fix warnings
+- optimization
+
+master ():
+
+- FOAF efficiency enhancements for fast nodes
+- gif filter
+- ssl fixes
+- logger fix
+- spare bitmap efficiency
+- reduce custom code
+- show semi-persistent update info next to bookmarks
```

Comment:
This isn't true: The update notifications *are* fully persistent across restart. That's the main point of one of the two PRs after all - previously when restarting you would lose *all* update notifications. Inlining the notifications into the bookmark list was the other PR.
Thus I recommend to change the wording and split up into two entries like this:
- persist notifications about updated bookmarks across restarts
- show notifications about updated bookmarks in the bookmark list itself, not only on the notifications page